spec/controllers/coronavirus/github_changes_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe Coronavirus::GithubChangesController do
+  include CoronavirusFeatureSteps
+
+  let(:stub_user) { create :user, :coronovirus_editor, name: "Name Surname" }
+  let(:coronavirus_page) { create(:coronavirus_page, slug: "landing") }
+
+  before do
+    stub_coronavirus_landing_page_content(coronavirus_page)
+    stub_coronavirus_publishing_api
+  end
+
+  describe "GET /coronavirus/github_changes/:slug" do
+    it "renders page successfuly" do
+      get :show, params: { slug: coronavirus_page.slug }
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it "does not render successfully if the user does not have Coronavirus editor permissions" do
+      stub_user.permissions = %w[signin]
+      get :show, params: { slug: coronavirus_page.slug }
+
+      expect(response).to have_http_status(:forbidden)
+    end
+  end
+
+  describe "PUT /coronavirus/github_changes/:slug" do
+    it "updates page content in publishing-api" do
+      put :update, params: { slug: coronavirus_page.slug }
+
+      assert_publishing_api_put_content(coronavirus_page.content_id)
```

Comment:
I assume these are copied over from before rather than your additions so I'm sorry for making a comment. I just feel this and the publish calls could probably benefit from an assertion on status code too (in this same test)

Reply:
I had those expectations, and took them out. I'll put them back in tomorrow.